src/Microsoft.DotNet.Darc/src/Darc/Options/GetDependencyGraphCommandLineOptions.cs

```diff
@@ -30,15 +31,22 @@ internal class GetDependencyGraphCommandLineOptions : CommandLineOptions
            + @"https://github.com/dotnet/corefx,C:\repos\corefx.")]
         public IEnumerable<string> RemotesMap { get; set; }
 
-        [Option('f', "flat", SetName = "output", HelpText = @"Returns a unique set of repository+sha combination.")]
+        [Option('f', "flat", HelpText = @"Returns a unique set of repository+sha combination.")]
         public bool Flat { get; set; }
 
-        [Option("graphviz", SetName = "output", HelpText = @"Returns the repository graph in GraphViz form.")]
-        public bool GraphViz { get; set; }
+        [Option("graphviz", HelpText = @"Writes the repository graph in GraphViz form, in the specified file")]
```

Comment:
>graphviz [](start = 17, length = 8)

The other options have - between the words. Should should this be graph-vizualization/graph-viz?

Reply:
graphviz is the product, so nope.